CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/BotData.cs

```diff
@@ -65,7 +63,7 @@ public interface IBotDataStore<T>
         /// <param name="botStoreType"> The bot store type.</param>
         /// <param name="cancellationToken"> The cancellation token.</param>
         /// <returns>Bot record that is stored for this key, or "empty" bot record ready to be stored</returns>
-        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken);
+        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
> = default(CancellationToken) [](start = 102, length = 29)

default arguments on interfaces are odd because they're not applied consistently to the derived class.

in addition, making CancellationToken optional encourages people to break cooperative cancellation by not passing the token.